emcc.py

```diff
@@ -2319,6 +2311,23 @@ def do_binaryen(final, target, asm_target, options, memfile, wasm_binary_target,
           passes.append('minifyWhitespace')
         final = shared.Building.js_optimizer_no_asmjs(final, passes)
       if DEBUG: save_intermediate('postclean', 'js')
+  # replace placeholder strings with correct subresource locations
```

Comment:
could this entire new code section be behind `if SINGLE_FILE` to avoid the overhead in the other case?